clients/client-cloudsearch/protocols/Aws_query.ts

```diff
@@ -4902,10 +4902,13 @@ const deserializeAws_queryDomainNameMap = (
   output: any,
   context: __SerdeContext
 ): { [key: string]: string } => {
-  return output.reduce((acc: any, pair: any) => {
-    acc[pair["key"]] = pair["value"];
-    return acc;
-  }, {});
+  return output.reduce(
+    (acc: any, pair: any) => ({
+      ...acc,
+      [pair["key"]]: pair["value"]
+    }),
+    {}
```

Comment:
Are these extra new lines needed? Formatting looks strange.

Reply:
{.... } notation is awesome

Reply:
> Are these extra new lines needed? Formatting looks strange.

Introduced by prettier